diff --git a/src/fixes/glint.js b/src/fixes/glint.js
index a4712a4b8513f384b0d15b15c4ea80dfc4fcccd8..1a8a5a9740a88cc8690c9470a7411396dcf68808 100644
--- a/src/fixes/glint.js
+++ b/src/fixes/glint.js
@@ -32,6 +32,13 @@ export function fixGTSExtensions(contents) {
       path.node.source.value = path.node.source.value.replace(/\.gts$/, '');
     });
 
+  root
+    .find(j.TSImportType)
+    .filter((path) => path.node.argument.value.includes('.gts'))
+    .forEach((path) => {
+      path.node.argument.value = path.node.argument.value.replace(/\.gts$/, '');
+    });
+
   return root.toSource();
 }
 
